backend/src/services/demux/updaters/blog/index.js

```diff
@@ -1,11 +1,11 @@
-const createPost = require('./createpost')
-const deletePost = require('./deletepost')
-const editPost = require('./editpost')
-const likePost = require('./likepost')
+import createPost from './createpost'
+import deletePost from './deletepost'
+import editPost from './editpost'
+import likePost from './likepost'
 
 const account = process.env.EOSIO_CONTRACT_ACCOUNT
 
-module.exports = [
+export default [
   {
     actionType: `${account}::createpost`, // account::action name
```

Comment:
Should this action name be camel case?

Reply:
When i initial wrote the contract code i followed the action naming conventions in a few sample contracts that i had found, but it looks like snake case is pretty popular. I may eventually switch to that, but definitely think that's outside of the scope of this PR